.github/workflows/backport.yml

```
@@ -17,6 +17,6 @@ jobs:
         run: |
           backport-assistant backport
         env:
-          BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.\\w+)"
+          BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.[\\w\\+]+)"
```

I think this would include `\` as a valid char.  Maybe instead 

```suggestion
          BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.[+\\w]+)"
```

(putting the `+` first just to avoid confusing people into thinking it applies to the `\\w`)